test/base/test_buffer.py

```diff
@@ -67,10 +67,10 @@ def test_stack(size=5, bufsize=9, stack_num=4):
         if done:
             obs = env.reset(1)
     indice = np.arange(len(buf))
-    assert np.allclose(buf.get(indice, 'obs'), np.array([
-        [1, 1, 1, 2], [1, 1, 2, 3], [1, 2, 3, 4],
-        [1, 1, 1, 1], [1, 1, 1, 2], [1, 1, 2, 3],
-        [3, 3, 3, 3], [3, 3, 3, 4], [1, 1, 1, 1]]))
+    assert np.allclose(buf.get(indice, 'obs'), np.expand_dims(
+        [[1, 1, 1, 2], [1, 1, 2, 3], [1, 2, 3, 4],
+         [1, 1, 1, 1], [1, 1, 1, 2], [1, 1, 2, 3],
+         [3, 3, 3, 3], [3, 3, 3, 4], [1, 1, 1, 1]], axis=-1))
```

Comment:
Is there a better way to do this? I see many ``expand_dims ``. Same below.

Reply:
This is what @Trinkle23897 suggested,  any suggestions?

Reply:
Since using an int as a state is not valid in a standard gym.Env, the state should be a np.ndarray including only 1 element: `np.array([1])`.